frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/enterprise.unit.spec.tsx

```diff
@@ -48,7 +48,14 @@ describe("SettingsEditor", () => {
     });
 
     await userEvent.click(screen.getByText("Embedding"));
-    await userEvent.click(screen.getByText("Interactive embedding"));
+    expect(
+      within(
+        screen.getByRole("article", {
+          name: "Interactive embedding",
+        }),
+      ).getByRole("link", { name: "Learn More" }),
```

Comment:
The test was wrong, like I've mentioned in one previous comment, we can't really click this label inside the interactive embedding card.